src/services/editNotesService.js

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+export const editNotesService = async (editnotes, encodedToken) => {
+  try {
+    const resp = await axios.post(
+      `/api/notes/${editnotes._id}`,
+      { note: editnotes },
+      {
+        headers: {
+          authorization: encodedToken,
+        },
+      }
+    );
+    return resp;
+  } catch (error) {
+    console.log(error.message);
```

Comment:
you can use toastify to show error on view

Reply:
I will add  this